### DISCUSSION TITLE
R-universe vs GitHub

### DISCUSSION THREAD

**Discussion by howardbaik** [r-universe-org/help#302] [General]
This R-universe project seems like a very cool project that I am looking into. Since this is branded as a "personal package repository", I was wondering what the difference is between this and GitHub, where we can just publish R packages free-for-all. 

I'm hoping to get opinions of people who are on R-universe and what they found the difference is with GitHub.

**Comment by jeroen** [comment]
It is not really either-or: r-universe is built on top of git(hub) to provide R specific services and infrastructure. For most R packages in R-universe, the source code is continuously updated from an upstream github repository, and then from there we build and deploy all the things to r-universe. To get a package on r-universe, you first need to put it on GitHub or another Git server.

**Comment by howardbaik** [comment]
Thank you for the explanation!
